packages/hasher/src/hashOfFiles.ts

```diff
@@ -21,20 +31,38 @@ export async function generateHashOfFiles(
 
   files.sort((a, b) => a.path.localeCompare(b.path));
 
-  const hashes = await Promise.all(
-    files.map(async file => {
-      const hasher = crypto.createHash("sha1");
-      hasher.update(file.path);
+  const hashes: string[] = [];
 
-      if (!file.dirent.isDirectory()) {
-        const fileBuffer = await fs.readFile(path.join(packageRoot, file.path));
-        const data = fileBuffer.toString().replace(newline, LF);
-        hasher.update(data);
+  const repoFileHashes = repoHashes.get(gitRoot)!;
+
+  for (const entry of files) {
```

Comment:
@VincentBailly I still think your PR is relevant since this PR isn't changing what files we are trying to get a hash for.

Reply:
Correct, I don't think @VincentBailly should drop that PR. This PR is meant to be complementary.